packages/namekit-react/src/components/Link.tsx

```diff
@@ -27,30 +31,41 @@ const isInternalLink = (href: string | undefined) => {
   return href.startsWith("/") || href.startsWith("#") || !href.includes("://");
 };
 
-export const Link: React.FC<LinkProps> = ({
+export const ExternalIcon: React.FC = () => <span>↗</span>;
```

Comment:
Perhaps we should add some horizontal spacing before / after this depending on it's placement? Right now it feels too close to the anchor text.

Reply:
@lightwalker-eth yeah, was going to address if this was the right direction 😄